### PR TITLE
Fix gRPC for mint annual-provisions

### PR DESCRIPTION
**Pull request by amaury1093**
### Description

This PR fixes the error on CLI command `q mint annual-provisions`. It may affect other commands that deal with sdk.Dec.

It is related to https://github.com/cosmos/cosmos-sdk/issues/8392.

### To Test:

Run a gaiad node (doesn't need to be with the hub genesis, an dummy genesis works too).

Then run the `q mint annual-provisions` CLI command.

### Before

```bash
./build/gaiad q mint annual-provisions           
Error: panic message redacted to hide potentially sensitive system info: panic
--snip--
```

### After

```bash
./build/gaiad q mint annual-provisions
1300006837.287849604578641445
```

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/main/README.md#merging-a-pr))

### CODE REVIEW

**Review by tac0turtle — APPROVED** [review]
utACK. Should we check other cmds?

**Comment by zmanian** [comment]
Thank you!

**Review by zmanian — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -17,4 +17,6 @@ require (
 	github.com/tendermint/tm-db v0.6.4
 )
 
+replace google.golang.org/grpc => google.golang.org/grpc v1.33.2
```
Comment (alessio) [inline go.mod:20]: I suspect we would need to add this to gaia's `go.mod` too @shahankhatch

**Review by alessio — APPROVED** [review]

**Comment by alessio** [comment]
@marbar3778 the docker hub action is complaining :-/

**Comment by shahankhatch** [comment]
> @marbar3778 the docker hub action is complaining :-/

I suspect this is because of Amaury's membership. See the PR that works for this cherry-picked commit https://github.com/cosmos/gaia/pull/650. I'm going to merge in and we can add Amaury to a team.

**Pull request merged**